omod/src/main/java/org/openmrs/module/coreapps/page/controller/findpatient/FindPatientPageController.java

```diff
@@ -28,6 +31,7 @@ public void get(PageModel model, @RequestParam("app") AppDescriptor app, UiSessi
         model.addAttribute("heading", app.getConfig().get("heading").getTextValue());
         model.addAttribute("label", app.getConfig().get("label").getTextValue());
         model.addAttribute("showLastViewedPatients", app.getConfig().get("showLastViewedPatients").getBooleanValue());
+		model.addAttribute("showRegistrationLink", app.getConfig().get("showRegistrationLink").getTextValue());
```

Comment:
@wluyima There is only one place, in the registrationButtonAppId configuration of the Find Patient app. If this empty or null, then the button is not displayed.

I am thinking that there is no need to update registration core - since this app configuration option solves all problems